pynest/examples/eprop_plasticity/eprop_supervised_regression_infinite-loop.py

```diff
@@ -275,43 +275,39 @@
 weights_out_rec = np.array(np.random.randn(n_rec, n_out) / np.sqrt(n_rec), dtype=dtype_weights)
 
 params_common_syn_eprop = {
-    "adam_beta1": 0.9,  # exponential decay rate for 1st moment estimate of Adam optimizer
-    "adam_beta2": 0.999,  # exponential decay rate for 2nd moment raw estimate of Adam optimizer
-    "adam_epsilon": 1e-8,  # small numerical stabilization constant of Adam optimizer
+    "optimizer": {
+        "type": "adam",  # algorithm to optimize the weights
+        "batch_size": n_batch,
+        "eta": 5e-3,  # learning rate
+        "Wmin": -100.0,  # pA, minimal limit of the synaptic weights
+        "Wmax": 100.0,  # pA, maximal limit of the synaptic weights
+        "beta_1": 0.9,  # exponential decay rate for 1st moment estimate of Adam optimizer
+        "beta_2": 0.999,  # exponential decay rate for 2nd moment raw estimate of Adam optimizer
+        "epsilon": 1e-8,  # small numerical stabilization constant of Adam optimizer
```

Comment:
```suggestion
        "batch_size": n_batch,
        "beta_1": 0.9,  # exponential decay rate for 1st moment estimate of Adam optimizer
        "beta_2": 0.999,  # exponential decay rate for 2nd moment raw estimate of Adam optimizer
        "epsilon": 1e-8,  # small numerical stabilization constant of Adam optimizer
        "eta": 5e-3,  # learning rate
        "Wmax": 100.0,  # pA, maximal limit of the synaptic weights
        "Wmin": -100.0,  # pA, minimal limit of the synaptic weights
```
The suggested change sorts the parameters after the `type` alphabetically.